imageRowIncrease: 1
imageColIncrease: 1
cameraRealiableDis: 8 
pointCloudLeafSize: 0.2

start_frame: 11391 #11400
end_frame: 11862 #13252
run_mode: 0
mapSave: true
data_folder: "/home/zjl/semantic_3d_avp_ws/src/semantic_3d_avp/data/map_yt/"
scene: 1  #yt 1 zjg1 2 zjg2 3 yq 4
save_directory: "/home/zjl/semantic_3d_avp_ws/src/semantic_3d_avp/data/slam_01/"
mapSaveLocation: "/home/zjl/semantic_3d_avp_ws/src/semantic_3d_avp/data/slam_01/slam_01.pcd" #-aligen-trans0.4

odomKeyFramDisThresh: 0.1
odomKeyFramAngThresh: 5
InvalidColorThresh: 60
#IcpMaximumIterations最大迭代次数 IcpMaxCorrespondenceDistance搜索对应点的最大距离 IcpTransformationEpsilon收敛条件，当前位姿与上次位姿插值 IcpEuclideanFitnessEpsilon均方误差和
useICP: false
IcpMaxCorrespondenceDistance: 10 #10 1
IcpMaximumIterations: 100
IcpTransformationEpsilon: 1e-8 #1e-8
IcpEuclideanFitnessEpsilon: 1e-3 #0.01 #0.01
icpFitnessScoreThresh: 2000
useNDT: true
usevins: true
uselidar: false
myuseNDT_ICP: false
useplane: true
usemyICP: false
#ndtResolution网格大小 
ndtTransformationEpsilon: 0.01
ndtResolution: 1.0
ndtFitnessScoreThresh: 10
useNDT_ICP: true
useNDT_GICP: false
useNDT_PICP: false
use_sac_ndt: false
useglobalfilter: true
imagecutrow: 0
imagecutrow1: 0
imagecutcol: 0
refacity: false
usewhite: false
transfer_surround: true
dynacut: false
thre_low: -4
thre_high: 4
tx: 0
ty: -0.4 
tz: 0
troll: 0
tpitch: 0
tyaw: 0.05

#143.407 52.5921 0 -0.000559854 -0.0131479 -0.590205 0.807146
init_x: 143.407
init_y: 52.5921
init_z: 0
init_rx: -0.000559854
init_ry: -0.0131479
init_rz: -0.590205
init_rw: 0.807146